cloud-regionsrv-client.spec

```diff
@@ -88,7 +88,7 @@ Requires:       cloud-regionsrv-client >= 6.0.0
 Guest registration plugin for images intended for Amazon EC2
 
 %package plugin-azure
-Version:        1.0.1
+Version:        1.1.0
```

Comment:
Should be 2.0.0 we are changing the output in an incompatible way.

Reply:
Done